build/protobuf.mk

```diff
@@ -17,15 +17,14 @@
 # This file is evaluated in the repo's parent directory. See main.go's
 # go:generate invocation.
 
-ORG_ROOT       := .
-REPO_ROOT      := $(ORG_ROOT)/cockroach
-GITHUB_ROOT    := $(ORG_ROOT)/..
+REPO_ROOT      := ./cockroach
+GITHUB_ROOT    := $(REPO_ROOT)/vendor/github.com
```

Comment:
scratch that last comment -- executing in .. is needed so our imports of `cockroach/blah` work as expected. I'll bring back `ORG_ROOT` and maybe rename it to `REPO_PARENT` for clarity.